cli/internal/cmd/apply.go

```diff
@@ -517,10 +530,18 @@ func (a *applyCmd) validateInputs(cmd *cobra.Command, configFetcher attestationc
 		cmd.PrintErrln("WARNING: Attestation temporarily relies on AWS nitroTPM. See https://docs.edgeless.systems/constellation/workflows/config#choosing-a-vm-type for more information.")
 	}
 
+	if !a.flags.skipPhases.contains(skipInitPhase) {
```

Comment:
nit: is it just me, but I find this easier to read;
skip suggests we should skip it. (feel free to ignore)
```suggestion
	if !a.flags.skipPhases.skip(skipInitPhase) {
```